recipes/ukpopulation/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "ukpopulation" %}
+{% set version = "1.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/06/4e/bd7d79d83a0b5535dabfec2bc245bec1a30cf918c4dd2bba336a3c4d1a7c/{{ name }}-{{ version }}.tar.gz
```

Comment:
Use the canonical url:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L14